### PR TITLE
update macros for VUB

### PR DESCRIPTION
**Pull request by smoors**

### CODE REVIEW

File: sites/brussel/macros.tex [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 \macro{\hpc}{VUB-HPC}
 \macro{\hpcInfra}{VUB-HPC}
 \macro{\hpcTeam}{VUB-HPC}
 \macro{\hpcname}{hydra}
 \macro{\loginnode}{hydra.vub.ac.be}
-\macro{\loginhost}{nic111.hydra.vub.ac.be}
+\macro{\loginhost}{nic50.hydra.vub.ac.be}
 \macro{\loginfingerprint}{41:68:9b:c2:f0:1c:d3:62:14:b6:71:f0:36:8a:7d:ec}
-\macro{\operatingsystem}{Scientific Linux release 6.4 (Carbon)}
+\macro{\operatingsystem}{CentOS Linux release 7.3}
```
Comment (wpoely86) [inline sites/brussel/macros.tex]: It's going to 7.4 next week.

Comment (smoors) [reply sites/brussel/macros.tex]: Yes, next week...

File: sites/brussel/macros.tex [hunk 2]
```diff
@@ -27,6 +27,6 @@
 \macro{\computenode}{r5e3cn13.hydra.brussel.vsc}
 \macro{\computenodeshort}{r4e2cn03}
```
Comment (wpoely86) [inline sites/brussel/macros.tex]: change these too

File: sites/brussel/macros.tex [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+\macro{\association}{Universitaire Associatie Brussel}
 % HPC definitions
 \macro{\hpc}{VUB-HPC}
 \macro{\hpcInfra}{VUB-HPC}
 \macro{\hpcTeam}{VUB-HPC}
 \macro{\hpcname}{hydra}
 \macro{\loginnode}{hydra.vub.ac.be}
-\macro{\loginhost}{nic111.hydra.vub.ac.be}
+\macro{\loginhost}{nic50.hydra.vub.ac.be}
 \macro{\loginfingerprint}{41:68:9b:c2:f0:1c:d3:62:14:b6:71:f0:36:8a:7d:ec}
```
Comment (wpoely86) [inline sites/brussel/macros.tex]: please verify this one.

File: sites/brussel/macros.tex (hunk 1 above)
Comment (wpoely86) [reply sites/brussel/macros.tex]: Well, I'm not merging your PR this week 😉

File: sites/brussel/macros.tex (hunk 3 above)
Comment (smoors) [reply sites/brussel/macros.tex]: this is just the name of a login node

File: sites/brussel/macros.tex (hunk 1 above)
Comment (smoors) [reply sites/brussel/macros.tex]: fine :)

File: sites/brussel/macros.tex (hunk 3 above)
Comment (wpoely86) [reply sites/brussel/macros.tex]: the fingerprint I mean.

File: sites/brussel/macros.tex (hunk 2 above)
Comment (smoors) [reply sites/brussel/macros.tex]: but then the pictures have to be changed as well. I am not ready to do that yet.

File: sites/brussel/macros.tex (hunk 3 above)
Comment (smoors) [reply sites/brussel/macros.tex]: I checked, it is still the same

**Comment by boegel** [comment]
@smoors You'll need to run `./check-version.sh intro-HPC` and commit the version update to make Travis happy, before this can be merged.

**Pull request merged**